Wrapping/Generators/CastXML/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ else()
       CONFIGURE_COMMAND ""
       BUILD_COMMAND ""
       INSTALL_COMMAND "${CMAKE_COMMAND}" -E copy_directory "${CMAKE_CURRENT_BINARY_DIR}/castxml-prefix/src/castxml" "${CMAKE_CURRENT_BINARY_DIR}/castxml"
+      DOWNLOAD_EXTRACT_TIMESTAMP FALSE
```

Comment:
CastXML is a CMake project, so the time stamp dependencies issues with configure/autoconf are not an issue.

Reply:
There is a warning associated to that line:
```
Wrapping/Generators/CastXML/CMakeLists.txt:63 (ExternalProject_Add)
```

Reply:
Yes, for this project likely either behavior of the timestamp should be OK. However, I thought there was motivation to use the NEW CMP135 behavior where it could be used, like the castXML project. To do that the CMP135 could be set to new, and this line removed, or the value could be set to TRUE.

Reply:
Done. Resolving. Let's see what the CIs tell us.